mantidproject/mantid
# ISIS Reflectometry: Save and Load individual batches

**Description of work.**
This will allow users to Save and Load individual batches for the ISIS Reflectometry GUI.



**To test:**
- Open Workbench
- Open ISIS Reflectometry
- Do some things and change some settings
- Click Batch->Save and save it somewhere
- Close the ISIS Reflectometry GUI
- Open it again and click Batch->Load
- All settings should be back and the table should be in a close but not completely the same state, which is unprocessed.

- Repeat for MantidPlot as they have different backbones for file writing.



Fixes #18826. 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
